docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -9,7 +9,7 @@
 
 `TLSUV_DEBUG=4` - set the log level of the underlying libuv library, higher is more verbose (level 4 means DEBUG)
 
-For more information about configuring the underlying Ziti C SDK with environment variables, see [the Ziti C SDK documentation](../../../../docs/reference/developer/sdk/ziti-sdk-c).
+For more information about configuring the underlying Ziti C SDK with environment variables, see [the Ziti C SDK documentation](/reference/developer/sdk/ziti-sdk-c.mdx).
```

Comment:
i think the /docs is needed here

Reply:
The `/docs` part of the URL is never needed when using absolute file references because they're relative to the root dir of the particular instance of the docs plugin.

Reply:
or maybe no .mdx? all i know is the link is dead on the preview site :)

Reply:
you need to revert this for "docusaurus reasons". I see that it works properly on the live site but not here on this preview... :/